kclvm/runner/src/custom_manifests_data/schema.stdout.golden

```diff
@@ -0,0 +1,5 @@
+age: 1
```

Comment:
Need to use serde_yaml needs expand Indexmap's YAML-ordered output capability. cc @NeverRaR

Reply:
> Need to use serde_yaml needs expand Indexmap's YAML-ordered output capability. cc @NeverRaR

fixed in https://github.com/KusionStack/KCLVM/pull/309